### PR TITLE
Sfl fixes

### PR DESCRIPTION
**Pull request by sndrs**
- make tag page items saveable
- don't show save count on user icon if there are none

screen shot 2015-06-19 at 12 57 31
screen shot 2015-06-19 at 12 56 56
screen shot 2015-06-19 at 12 59 10

### CODE REVIEW

**Comment by rich-nguyen** [comment]
:+1:

**Comment by uplne** [comment]
Looks good :+1:

File: static/src/javascripts/projects/common/modules/save-for-later.js [hunk 1]
```diff
@@ -360,7 +360,11 @@ define([
 
             var profile = $('.brand-bar__item--profile');
             profile.addClass('brand-bar__item--profile--show-saved');
-            $('.control__icon-wrapper', profile).attr('data-saved-content-count', count);
+            if (count > 0) {
+                $('.control__icon-wrapper', profile).attr('data-saved-content-count', count);
+            } else {
+                $('.control__icon-wrapper', profile).removeAttr('data-saved-content-count');
```
Comment (OliverJAsh) [inline static/src/javascripts/projects/common/modules/save-for-later.js]: This leads to:

image

I think we need to make the line above only run if the count `> 0`.

Comment (OliverJAsh) [inline static/src/javascripts/projects/common/modules/save-for-later.js]: My bad, this is before your change was merged AFAICS. Never mind!

**Comment by sndrs** [comment]
does that happen if you rebuild the css? https://github.com/guardian/frontend/pull/9552/files#diff-aa84528ce5482196c6b40210c84470abR217 should take care of that?

**Comment by OliverJAsh** [comment]
:+1:

**Pull request merged**